PyTorch_dcgan_lsun/README.md

```diff
@@ -0,0 +1,50 @@
+# PyTorch - DCGAN
+
+This example is based on [this implementation](https://github.com/pytorch/examples/tree/master/dcgan)
```

Comment:
Maybe providing the proper git clone command would be more handy for the user?

Reply:
BTW a command for downloading only this example, instead of the whole PyTorch examples repo

Reply:
As far I was able to find out it isn't possible to clone just one subdirectory of a non-local git repository

Reply:
done